python/desc/twinkles/sqlite_tools.py

```diff
@@ -29,3 +31,30 @@ def create(self, columns, table, condition=None):
             query += ' ' + condition
         data = [row for row in self.conn.execute(query)]
         return pd.DataFrame(data=data, columns=columns)
+
```

Comment:
@jchiang87 One different thing I was worried about:  Should we try to drop duplicates, I think this query could pull in two records for the same pointing with two different propIDs. (This is taken care of in the script with a `DISTINCT` in the query, but I don't see it in the dataFrames). Also if `propID` is  becomes a requested column, a simple `DISTINCT` will not do as the propIDs are different.